src/FSharp.Markdown/Markdown.fs

```diff
@@ -54,17 +55,17 @@ and MarkdownEmbedSpans =
 /// Paragraphs are headings, inline paragraphs, code blocks, lists, quotations, tables and
 /// also embedded LaTeX blocks.
 type MarkdownParagraph = 
-  | Heading of int * MarkdownSpans
-  | Paragraph of MarkdownSpans
-  | CodeBlock of string * string * string
-  | InlineBlock of string
-  | ListBlock of MarkdownListKind * list<MarkdownParagraphs>
-  | QuotedBlock of MarkdownParagraphs
-  | Span of MarkdownSpans
-  | LatexBlock of list<string>
-  | HorizontalRule of char
-  | TableBlock of option<MarkdownTableRow> * list<MarkdownColumnAlignment> * list<MarkdownTableRow>
-  | EmbedParagraphs of MarkdownEmbedParagraphs
+  | Heading of n:int * spans:MarkdownSpans * range:MarkdownRange option
```

Comment:
I think `size` is better name than `n` - and as above, I prefer `body` over `spans` for `Heading`, `Paragraph` and `Span`.